<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit_calc</class>
 <widget class="QDialog" name="credit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit_Calc</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">
	background-color: rgb(22, 29, 39);
	color: rgb(68, 235, 153); /* Цвет текста */</string>
  </property>
  <widget class="QRadioButton" name="radioButton_annuity">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>17</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
	background-color: rgb(22, 29, 39);
    border-bottom: 3px solid rgb(123, 27, 209);
	color: rgb(68, 235, 153); 
}</string>
   </property>
   <property name="text">
    <string>Annuity</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_different">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>17</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
	background-color: rgb(22, 29, 39);
    border-bottom: 3px solid rgb(123, 27, 209);
	color: rgb(68, 235, 153); 
}</string>
   </property>
   <property name="text">
    <string>Differentiated</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_credit_sum">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>77</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(123, 27, 209);</string>
   </property>
   <property name="minimum">
    <double>10.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000000000013287555072.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>10000.000000000000000</double>
   </property>
   <property name="value">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_credit_sum">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>57</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(22, 29, 39);
	border: 1px solid rgb(22, 29, 39); /* Цвет рамки */
	color: rgb(68, 235, 153); /* Цвет текста */
}</string>
   </property>
   <property name="text">
    <string>Total credit amount (RUB):</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_term">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>137</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(123, 27, 209);</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_term">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>117</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(22, 29, 39);
	border: 1px solid rgb(22, 29, 39); /* Цвет рамки */
	color: rgb(68, 235, 153); /* Цвет текста */
}</string>
   </property>
   <property name="text">
    <string>Term:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_term">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>137</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Month</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Year</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_bid">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>177</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(22, 29, 39);
	border: 1px solid rgb(22, 29, 39); /* Цвет рамки */
	color: rgb(68, 235, 153); /* Цвет текста */
}</string>
   </property>
   <property name="text">
    <string>Interest rate:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_bid">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>197</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(123, 27, 209);</string>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>10000000000000.000000000000000</double>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
   </property>
   <property name="value">
    <double>12.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>237</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(123,27,209);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}
</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_overpayment">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(22, 29, 39);
	border-bottom: 1px solid rgb(123, 27, 209); /* Цвет рамки */
	color: rgb(68, 235, 153); /* Цвет текста */
}</string>
   </property>
   <property name="text">
    <string>Overpayment:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>431</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(22, 29, 39);
	border-bottom: 1px solid rgb(123, 27, 209); /* Цвет рамки */
	color: rgb(68, 235, 153); /* Цвет текста */
}</string>
   </property>
   <property name="text">
    <string>Total payment by credit:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>661</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView{
	background-color: rgb(22, 29, 39);
border: 1px solid rgb(22, 29, 39); /* Цвет рамки */
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_overpayment_result">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>400</y>
     <width>641</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(22, 29, 39);
	border: 1px solid rgb(22, 29, 39); /* Цвет рамки */ 
	color: rgb(68, 235, 153); /* Цвет текста */
}</string>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_result">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>430</y>
     <width>601</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string>QLabel{
	background-color: rgb(22, 29, 39);
	border: 1px solid rgb(22, 29, 39); /* Цвет рамки */ 
	color: rgb(68, 235, 153); /* Цвет текста */
}</string>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 14pt &quot;Sans Serif&quot;;
	background-color: rgb(22, 29, 39);
    border-left: 2px solid rgb(123, 27, 209);
	color: rgb(68, 235, 153); 
}</string>
   </property>
   <property name="text">
    <string>  Payment schedule:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(22, 29, 39);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>331</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView{
	
	background-image: url(:/img/w.PNG);
	background-color: rgb(22, 29, 39);
border: 1px solid rgb(22, 29, 39); /* Цвет рамки */
}</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_3">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>10</y>
     <width>1171</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView{
	
	background-image: url(:/img/ba.PNG);
	background-color: rgb(22, 29, 39);
border: 1px solid rgb(22, 29, 39); /* Цвет рамки */
}</string>
   </property>
  </widget>
  <zorder>tableView_3</zorder>
  <zorder>tableView_2</zorder>
  <zorder>radioButton_annuity</zorder>
  <zorder>radioButton_different</zorder>
  <zorder>doubleSpinBox_credit_sum</zorder>
  <zorder>label_credit_sum</zorder>
  <zorder>spinBox_term</zorder>
  <zorder>label_term</zorder>
  <zorder>comboBox_term</zorder>
  <zorder>label_bid</zorder>
  <zorder>doubleSpinBox_bid</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_overpayment</zorder>
  <zorder>label_total</zorder>
  <zorder>tableView</zorder>
  <zorder>label_overpayment_result</zorder>
  <zorder>label_total_result</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
